File: app/src/main/java/com/example/ordinacija/Data/UslugeResultVM.java
package com.example.ordinacija.Data;

import java.io.Serializable;
import java.util.List;

public class UslugeResultVM implements Serializable {
    public static class Row implements Serializable {
        public int UslugaID;
        public String Naziv;
        public String Opis;
    }
    public List<UslugeResultVM.Row> rows;
}

File: WebServisi/Ordinacija/Ordinacija_API/ModelVM/UslugeResultVM.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace Ordinacija_API.ModelVM
{
    public class UslugeResultVM
    {
        public List<Row> rows { get; set; }

        public class Row
        {
            public int UslugaID { get; set; }
            public string Naziv { get; set; }
            public string Opis { get; set; }


        }
    }
}